### PR TITLE
feat(semantictokens): подсветка платформенных member-методов через accessCall

### PR DESCRIPTION
**Pull request by nixel2007**
## Summary

- Новый `PlatformMemberMethodCallSemanticTokensSupplier` подсвечивает вызовы методов на типизированных переменных вида `receiver.method(...)` для платформенных типов как `Method+defaultLibrary`.
- Источник истины — `TypeService.findMemberAt(...)` (тот же путь, что hover/go-to-definition): вывод типа ресивера через `ExpressionTypeInferencer`, поиск member через `TypeRegistry.getMembers`.
- Source-defined вызовы (общие модули, OScript-library, локальные методы) остаются за `MethodCallSemanticTokensSupplier` — этот сапплаер пропускает позиции, уже зарегистрированные в `ReferenceIndex`, чтобы не дублировать токен.
- Async-модификатор для платформенных member-методов пока не выставляется: у `MemberDescriptor` нет поля `async`. Отдельная задача — расширение `PlatformMetadata` + парсера платформенных контекстов.
- Follow-up issue #3949 — симметричный сапплаер для accessProperty (`Стр.Имя` сейчас тоже не подсвечивается).

Закрывает баг из юзер-репорта «не красится метод после `Ждать`»: корень был в том, что `СертификатКриптографии.ИнициализироватьАсинх(...)` — это вызов метода платформенного типа, который ни один существующий сапплаер не покрывал.

## Test plan

- [x] `PlatformMemberMethodCallSemanticTokensSupplierTest`:
  - `testPlatformMethodOnTypedVariable` — `М.Добавить(...)` (Массив) → Method+DefaultLibrary
  - `testPlatformMethodAfterAwait` — `Ждать М.Добавить(...)` (юзер-репорт)
  - `testAccessPropertyIgnored` — `Стр.Имя` не подсвечивается (это для follow-up)
  - `testUnknownTypeReceiverProducesNoToken` — без типа пусто
- [x] `./gradlew test --tests "...semantictokens.*" --tests "...SemanticTokensProviderTest"` — зелёно, регрессий нет
- [x] `./gradlew bootJar` — собирается

## Draft

PR оформлен как draft: жду подтверждения подхода (через `TypeService.findMemberAt` против самостоятельного построения BslExpression) и решения по сопутствующему issue #3949 — публиковать оба супплаера вместе одним PR или отдельно.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Release Notes

* **New Features**
  * Enhanced semantic token recognition for method calls on typed variables, providing improved syntax highlighting with proper token classification.
  * Better support for semantic highlighting of platform method calls even when expressions follow await operations.

* **Tests**
  * Added comprehensive test coverage for semantic token behavior across multiple scenarios including typed variable method calls, post-await expressions, and edge cases with property access and untyped receivers.

[Review Change Stack](https://app.coderabbit.ai/change-stack/1c-syntax/bsl-language-server/pull/3950?utm_source=github_walkthrough&utm_medium=github&utm_campaign=change_stack)

### CODE REVIEW

File: src/test/java/com/github/_1c_syntax/bsl/languageserver/semantictokens/PlatformMemberMethodCallSemanticTokensSupplierTest.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Import;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@CleanupContextBeforeClassAndAfterClass
```
Comment (nixel2007) [inline src/test/java/com/github/_1c_syntax/bsl/languageserver/semantictokens/PlatformMemberMethodCallSemanticTokensSupplierTest.java]: зачем?

File: src/main/java/com/github/_1c_syntax/bsl/languageserver/semantictokens/PlatformMemberMethodCallSemanticTokensSupplier.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    var ast = documentContext.getAst();
+    if (ast == null) {
+      return entries;
+    }
+
+    // Позиции, уже обработанные MethodCallSemanticTokensSupplier: пропускаем,
+    // чтобы не дублировать токен на одной и той же позиции.
+    Set<Position> sourceDefinedCallSites = collectSourceDefinedCallSites(documentContext);
+
+    for (var node : Trees.findAllRuleNodes(ast, BSLParser.RULE_accessCall)) {
```
Comment (nixel2007) [inline src/main/java/com/github/_1c_syntax/bsl/languageserver/semantictokens/PlatformMemberMethodCallSemanticTokensSupplier.java]: findAllRuleNodes - generic method.
